pkg/logs/processor/processor.go

```diff
@@ -7,12 +7,15 @@ package processor
 
 import (
 	"fmt"
-	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/util"
+	log "github.com/cihub/seelog"
+	"strings"
```

Comment:
Most of the time, we group packages by builtin/external/internal

Reply:
👍